### ISSUE TITLE
Missing ionic-native

### ISSUE THREAD

**Issue by billygerhard** [driftyco/ionic-starter-super#85]
When creating a new project, the ionic CLI created the package.json with the new ionic-native code (@ionic-native/xxxx), but the code from the super starter still uses IMPORT {} FROM 'ionic-native' and causes issues during run time.

**Comment by jeffhongxiao** [comment]
i can confirm the same issue happens on my machine.

Details:
```
Stack
Error: Cannot find module "ionic-native"
    at v (http://localhost:8100/build/polyfills.js:3:4864)
    at Object.<anonymous> (http://localhost:8100/build/main.js:116830:7)
    at __webpack_require__ (http://localhost:8100/build/main.js:20:30)
    at Object.<anonymous> (http://localhost:8100/build/main.js:102085:73)
    at __webpack_require__ (http://localhost:8100/build/main.js:20:30)
    at Object.<anonymous> (http://localhost:8100/build/main.js:128721:70)
    at __webpack_require__ (http://localhost:8100/build/main.js:20:30)
    at http://localhost:8100/build/main.js:66:18
    at http://localhost:8100/build/main.js:69:10
```

Environment:
```
Ionic Framework: 2.3.0
Ionic App Scripts: 1.1.4
Angular Core: 2.4.8
Angular Compiler CLI: 2.4.8
Node: 6.10.0
OS Platform: Linux 4.4
Navigator Platform: Linux x86_64
User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36
```

**Comment by pwespi** [comment]
This is fixed in PR #84.
